### ISSUE TITLE
Snap and devmode

### ISSUE THREAD

**Issue by xeor** [trustcrypto/OnlyKey-App#133]
Installing the onlykey-app via snap now needs to be done with the `--devmode` flag for it to work.
This is noted at https://docs.crp.to/linux.html#install-app, and there is a forum thread https://forum.snapcraft.io/t/use-hidraw-interface-for-onlykey-app/13736 about this.

The issues refeered to by the forum-thread looks fixed and merged into snapd.

Can the discussion on the forum-thread continue? Or is there any other blockers?

**Comment by onlykey** [comment]
@xeor Yes the discussion can continue, last I checked there were no snaps that worked correctly with U2F devices. This should be changed by now, for our next app release I will see if we can get it working. Any help with this would be appreciated, or if you could point me to a snap that is known to work with U2F devices I could see what they did to make it work.

**Comment by xeor** [comment]
I'm new to snaps, but it looks like yubico is also using snaps. They have an authenticator there at least at https://snapcraft.io/yubioath-desktop. The snapcraft file is at https://github.com/Yubico/yubioath-desktop/blob/master/snap/snapcraft.yaml. Maybe that helps? I havent tried it, but it might give some clues. The docs for the `u2f-devices` interface for snap is at https://snapcraft.io/docs/u2f-devices-interface

**Comment by dnorthup-ums** [comment]
I presume this has been fixed? If so, is there a chance that the snaps app image will ever get updated?

**Comment by onlykey** [comment]
At this point we don't have a work around to get USB devices working with Snap except using the devmode flag.
